deployment-aws/terraform/storage.tf

```diff
@@ -41,3 +45,94 @@ resource "aws_s3_bucket_server_side_encryption_configuration" "cromwell_storage_
     bucket_key_enabled = true
   }
 }
+
+resource "aws_s3_bucket_policy" "logging_bucket_policy"  {
+  bucket = aws_s3_bucket.log_bucket.id
+  policy = data.aws_iam_policy_document.logging_bucket_policy_doc.json
```

Comment:
I think inline policy doc is perfectly fine here since this is what AWS docs provide